### PR TITLE
feat: setup this repo with the basics

### PR DESCRIPTION
**Pull request by Carlos-Muniz**
Basics were picked from edx-cookiecutters. 

This repo was made as part of the prototype for OEP-58.

### CODE REVIEW

**Review by feanil — COMMENTED** [review]
I don't see a `Makefile`.  Did you forget to add it?

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+metadata:
+  name: 'openedx-atlas'
+  description: "An Open edX CLI tool for moving translation files from openedx-translations."
+  links:
+  - url: "https://github.com/openedx/openedx-atlas"
+    title: "Source Code"
+    icon: "Code"
+  annotations:
+    openedx.org/arch-interest-groups: "Carlos-Muniz"
+    github.com/project-slug: openedx/openedx-atlas
```
Comment (feanil) [inline catalog-info.yaml:12]: You don't need this.

File: catalog-info.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'openedx-atlas'
+  description: "An Open edX CLI tool for moving translation files from openedx-translations."
+  links:
+  - url: "https://github.com/openedx/openedx-atlas"
```
Comment (feanil) [inline catalog-info.yaml:7]: You also don't need to add a link to the source, that gets added automatically.

File: catalog-info.yaml (hunk 1 above)
Comment (Carlos-Muniz) [reply catalog-info.yaml]: removed

File: catalog-info.yaml (hunk 2 above)
Comment (Carlos-Muniz) [reply catalog-info.yaml]: removed

**Comment by Carlos-Muniz** [comment]
@feanil I just added the missing Makefile

**Review by feanil — APPROVED** [review]

**Pull request merged**